test/calibration/test_calibrations.py

```diff
@@ -894,6 +909,33 @@ def test_single_control_channel(self):
 
         self.assertEqual(self.cals.get_schedule("tcp", (3, 2)), expected)
 
+    def test_inst_map_stays_consistent(self):
+        """Check that get schedule and inst map are in sync in a complex ECR case.
+
+        This needs to be in a subclass of CrossResonanceTest. Note, that this test
+        will fail if the coupling_map and the control_channel_map are not consistent
+        with each other. This is because the coupling_map is used to build the
+        _operated_qubits variable which determines the qubits of the instruction to
+        which a schedule is associated.
```

Comment:
I think this test is a useful one to have, but this description is too specific to the current implementation. To me the important thing that is being tested here is that when a parameter value is updated for a parameter that is used in a schedule nested inside a call instruction of an outer schedule that that outer schedule is also updated in the instruction schedule map. There are different problems that could cause that not to happen and this paragraph describes one specific case.